### ISSUE TITLE
Android TV Navigation Issue

### ISSUE THREAD

**Issue by camilo1505** [aniyomiorg/aniyomi#1006]
### Steps to reproduce

1. Install Animeyomi extension on Android TV.
2. Perform a global search.
3. Attempt to navigate downwards through the search results.
4. The selected component remains stuck on the search bar or the back button.
5. Unable to add any titles to the library.

### Expected behavior

After step 3, the selected component should allow the user to navigate through the search results and add titles to the library.

### Actual behavior

The selected component does not allow navigation within the global search results.

### Aniyomi version

Aniyomi Preview r6147

### Android version

Android 9 tv

### Device

Hyundai Android TV

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/jmir1/aniyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://aniyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.3.10](https://github.com/jmir1/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by juniornff** [comment]
The same happens to me. A temporary solution is to use a mouse connected to the TV or an application that simulates a mouse

**Comment by Quickdesh** [comment]
https://github.com/aniyomiorg/aniyomi/issues/162
